src/frontend/komponenter/Behandling/Inngangsvilkår/Aleneomsorg/Aleneomsorg.tsx

```diff
@@ -33,7 +33,7 @@ export const Aleneomsorg: React.FC<VilkårProps> = ({
                                     )}
                                     <AleneomsorgInfo
                                         barnMedSamvær={grunnlag.barnMedSamvær}
-                                        barnId={vurdering.barnId}
+                                        barnId={barn.barnId}
```

Comment:
Inte direkt kopplad till denna PR, men behøver man skicka med barnId till aleneomsorg? 
`grunnlag.barnMedSamvær` innehåller væl barnId? 
Och `barnMedSamvær` borde kunna få `barn`, som bara ær barnet så behøver man inte ha med 
`const gjeldendeBarn = barnMedSamvær.find((it) => it.barnId === barnId);` i AleneomsorgInfo

Reply:
JO! Var nog bara lathet från min sida [fikset her](https://github.com/navikt/familie-ef-sak-frontend/pull/227/commits/a3a39cdfe4cc098dbb09866e95c146fe2275b3e7)